_episodes/12-cluster.md

```diff
@@ -528,16 +109,26 @@ upgrade your OS.
 Individual computers that compose a cluster are typically called *nodes*
 (although you will also hear people call them *servers*, *computers* and
 *machines*). On a cluster, there are different types of nodes for different
-types of tasks. The node where you are right now is called the *head node*,
-*login node*, *landing pad*, or *submit node*. A login node serves as an access
+types of tasks. The node where you are right now is called the *login node*,
+*head node*, *landing pad*, or *submit node*. A login node serves as an access
 point to the cluster.
 
-As a gateway, it is well suited for uploading and downloading files, setting up
-software, and running quick tests. Generally speaking, the login node should
-not be used for time-consuming or resource-intensive tasks. You should be alert
-to this, and check with your site's operators or documentation for details of
-what is and isn't allowed. In these lessons, we will avoid running jobs on the
-head node.
+As a gateway, the login node should not be used for time-consuming or
+resource-intensive tasks. You should be alert to this, and check with your
+site's operators or documentation for details of what is and isn't allowed. It
+is well suited for uploading and downloading files, setting up software, and
+running tests. Generally speaking, in these lessons, we will avoid running jobs
```

Comment:
```suggestion
running short tests that are not resource intensive. Generally speaking, in these lessons, we will avoid running jobs
```

Reply:
This feels overly prescriptive: the criterion is not run-time or CPU/RAM usage, but rather impact on other users and the ability of the login node to do serve its primary role. As the central point of entry onto a system that may have a different architecture or software stack than a user's off-cluster resources, it is an appropriate place to build software and run test suites, some of which are time-consuming and perform a non-trivial amount of computational work, and that is right and proper.